Add: RhinoScript 내용정리

lee-siya opened the pull request:
RhinoScript 정리

https://developer.rhino3d.com/guides/scripting/

위 링크에 
ScriptEditor
Scrip_Component-Grasshopper
Grasshopper_Scripting-Python
세 내용을 정리했습니다.

lee-siya committed:
Add: RhinoScript 내용정리

lee-siya committed:
docs: ScriptEditor_Command Figure 추가

ScriptEditor_Command 파일에 이미지 추가했습니다.

lee-siya committed:
docs: Script_Component-Grasshopper.md Figure 추가

Script_Component-Grasshopper.md 파일에 이미지 추가했습니다.

lee-siya committed:
docs: Grasshopper_Scripting-Python.md Figure 추가

Grasshopper_Scripting-Python.md 파일에 이미지 추가했습니다.

kimyoo04 commented:
`유형 힌트` 라고 번역된 것은 `타입 힌트` 라고 적는 것이 이해하기 좋을 것 같습니다. 
전부 `타입 힌트` 로 고쳐주시면 감사하겠습니다.

kimyoo04 commented on RhinoScript/Grasshopper_Scripting-Python.md:
```diff
@@ -0,0 +1,733 @@
+
+![Figure_96](https://github.com/user-attachments/assets/81abaa33-919e-4d78-94ae-2dc0ae544b23)
+
+하단의 중단점 트레이에는 모든 중단점이 표시되며, 활성화/비활성화 또는 지우기 버튼이 제공됩니다:s
```
typo

lee-siya replied:
수정했습니다. 확인 감사합니다.

kimyoo04 commented on RhinoScript/Grasshopper_Scripting-Python.md:
```diff
@@ -0,0 +1,733 @@
+
+### Type Safety
+
+파이썬은 타입 안전 언어가 아닙니다. 즉, x라는 변수에 어떤 유형이든 값을 할당할 수 있습니다. 이는 변수 x가 int로 정의된 경우 Sphere 유형의 값을 할당할 수 없는 C#과 같은 타입이 지정된 언어와는 매우 다릅니다.
```
`유형` 이라고 번역된 부분은 전부 `타입` 으로 고쳐주시면 감사하겠습니다.

lee-siya replied:
수정했습니다.

kimyoo04 commented on RhinoScript/Grasshopper_Scripting-Python.md:
```diff
@@ -0,0 +1,733 @@
+
+필드와 속성이 있는 더 복잡한 데이터 유형의 경우 변수를 확장하여 해당 멤버의 현재 값을 확인할 수 있습니다:
+
+    Figure_104
```
이미지를 붙여넣는 방식이 잘못된 것일까요? 어떻게 붙여 넣었을까요?

lee-siya replied:
저렇게 Figure_104 적어둔 것은 저장해둔 이미지 파일 이름과 위치를 표시하기 위함입니다. 깃허브에서 이미지를 넣기 전 상태입니다.

lee-siya replied:
업로드 되지 않았던 이미지 전부 추가 했습니다.

kimyoo04 commented on RhinoScript/Grasshopper_Scripting-Python.md:
```diff
@@ -0,0 +1,733 @@
+
+    Figure_121
+
+스크립트 구성 요소를 Grasshopper 플러그인에 게시하는 방법은 Rhino와 Grasshopper 플러그인 만들기에서 확인하세요.
```
https://developer.rhino3d.com/guides/scripting/projects-publish/

링크 삽입해주시면 감사하겠습니다.

kimyoo04 replied:
[Creating Rhino Projects](https://developer.rhino3d.com/guides/scripting/projects-create/) 

링크도 참고로 넣어주시면 감사하겠습니다.

lee-siya replied:
알겠습니다. 추가하겠습니다.

kimyoo04 commented on RhinoScript/Grasshopper_Scripting-Python.md:
```diff
@@ -0,0 +1,733 @@
+
+![Figure_100](https://github.com/user-attachments/assets/f9d898d6-6eb7-4012-8795-f19c94b1ed23)
+
+왼쪽부터 차례대로 설명합니다:
++ **Continue**: 다른 중단점에서 멈출 때까지 스크립트를 계속 실행합니다.
++ **Step Over**: 현재 줄을 실행하고 다음 줄로 이동합니다.
++ **Step In**: 현재 줄에 함수 호출이 포함되어 있으면 함수 코드를 정의하는 줄로 이동합니다.
++ **Step Out**: 이전에 함수 코드에 들어간 경우 함수에서 호출 코드로 제어가 반환될 때까지 함수 코드를 계속 실행하고 거기서 멈춥니다.
++ **Stop**: 스크립트 디버깅을 중지하고 나머지 실행을 계속하지 않습니다.
+
```
3개의 파일에서 중복이 되는 내용으로 보입니다. 혹시 파일을 어떻게 구분해서 작성하신 것일까요?

lee-siya replied:
홈페이지에서 나오는 내용대로 정리한 것이긴 한데 중복내용 지우는게 나을까요?

kimyoo04 replied:
네~ 지우는게 좋아 보입니다. 아니면 어느 파일을 참조할지 적어줘도 좋을 것 같네요.

lee-siya committed:
docs: Script_Component-Grasshopper.md 타입, 타입 힌트 수정

Script_Component-Grasshopper 파일에서

유형 > 타입
유형 힌트 > 타입 힌트

로 수정했습니다.

lee-siya committed:
docs: ScriptEditor_Command.md 타입 수정

ScriptEditor_Command 파일에서

유형 > 타입

으로 수정했습니다.

lee-siya committed:
docs: Grasshopper_Scripting-Python.md 타입, 타입 힌트 수정

Grasshopper_Scripting-Python 파일에서

유형 > 타입
유형 힌트 > 타입 힌트

로 수정했습니다.

lee-siya committed:
docs: Grasshopper_Scripting-Python.md typo

확인해주신 오타 수정했습니다.

lee-siya committed:
docs: Grasshopper_Scripting-Python.md 이미지 추가, 링크 첨부

Grasshopper_Scripting-Python 파일에서 업로드 되지 않던 이미지를 추가했고 코멘트 주신 링크도 첨부했습니다.

스크립트 구성 요소를 Grasshopper 플러그인에 게시하는 방법은 Rhino와 Grasshopper 플러그인 만들기에서 확인하세요. <br>
[Creating Rhino Projects](https://developer.rhino3d.com/guides/scripting/projects-create/) <br>
[Creating Rhino and Grasshopper Plugins](https://developer.rhino3d.com/guides/scripting/projects-publish/)
